packages/webpack-config/src/helpers/paths.js

```diff
@@ -37,7 +37,9 @@ export const getCustomWebpackConfig = configName =>
       try {
         customWebpackConfig = require(customWebpackConfigPath);
       }
-      catch {
+      catch (err) {
+        console.log("Failed to load config file:");
+        console.error(err);
```

Comment:
maybe you could do this in one line using `console.error("Failed to load config file:", err);`

Reply:
we could also discuss, whether this could be a warning instead of an error, because there is a fallback